src/node.js

```diff
@@ -74,23 +78,27 @@ Node.fromValue = function fromValue (value) {
   return Literal.fromValue(value)
 }
 
-Node.toJS = function fromJS (term) {
+const Namespace = require('./namespace')
+const ns = { xsd: Namespace('http://www.w3.org/2001/XMLSchema#') }
+
+Node.toJS = function toJS (term) {
   if (term.elements) {
     return term.elements.map(Node.toJS) // Array node (not standard RDFJS)
   }
   if (!term.datatype) return term // Objects remain objects
-  if (term.datatype.equals(ns.xsd('boolean'))) {
+  if (term.datatype.sameTerm(ns.xsd('boolean'))) {
```

Comment:
Just wondering about why this changed, whereas RDF/JS is still using `equals`.

Reply:
I guess sameTerm I understand but equals looks like it should be the same as === or == which could require the object to be the same object ... not sure why I ended up changing it.   Happy either way

Reply:
I would recommend `equals`; it's not `===` but rather "do these objects represent the same thing", i.e., will they reasonably behave the same in all circumstances?